src/lib/compose.js

```diff
@@ -51,34 +74,39 @@ export const some: Composer = validators => data =>
 
 // Helper
 type Flatten = (Array<Validator>, Array<Validator>) => Array<Validator>;
-const flatten: Flatten = (acc, vs) => [...acc, ...vs];
+export const flatten: Flatten = (acc, vs) => [...acc, ...vs];
```

Comment:
I think we should put this in another file (with the other helpers maybe?) and change it to be a more generic array flatten method (i.e. `const flatten = arr1 => arr1.reduce(...)`) with more generic types (`Array<any>` or similar)